README.md

```diff
@@ -64,3 +64,14 @@ This can be run with test explorer or pytest
 You can install pytest via pip
 To run pytest, simply run the following command from the root of your project.
 $ pytest
+
+To build the docker image, run either the development or the production build
+$ docker build --target development --tag todo-app:dev .
+$ docker build --target production --tag todo-app:prod .
+
+To run the docker to create a development or production container from the image
+$ docker run --env-file ./.env -p 5000:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
+For production, the trello key and token is required in the command below
+$ docker run --env TRELLO_KEY=[TRELLO_KEY] --env TRELLO_TOKEN=[TRELLO_TOKEN] -d -p 127.0.0.1:5000:5000 todo-app:prod
```

Comment:
This doesn't quite work as written. You also need the board id (or pray they have chosen the correct board name!):
```suggestion
$ docker run --env TRELLO_KEY=[TRELLO_KEY] --env TRELLO_TOKEN=[TRELLO_TOKEN] --env TRELLO_BOARD_ID=[TRELLO_BOARD_ID] -d -p 127.0.0.1:5000:5000 todo-app:prod
```